kura/org.eclipse.kura.container.provider/OSGI-INF/metatype/org.eclipse.kura.container.provider.ContainerInstance.xml

```diff
@@ -123,6 +123,15 @@
             type="String" cardinality="1"
             required="false" default="" />
             
+        <AD id="container.runtime"
+            name="Runtime"
+            description="Specify the runtime the container can use."
```

Comment:
Specifies the fully qualified name of an alternate OCI-compatible runtime, which is used to run commands specified by the "run" instruction. Example: "nvidia" corresponds to "--runtime=nvidia".